src/controller/jobRoleController.ts

```diff
@@ -13,4 +13,7 @@ export namespace JobRoles {
 
         res.render("job-roles");
     }
+    export async function getIndex(req: Request, res: Response): Promise<void> {
```

Comment:
I would want to see this in it's own controller as it has nothing to do with job roles

Reply:
change made